### ISSUE TITLE
rename repo to seneca-concorda?

### ISSUE THREAD

**Issue by AdrianRossouw** [Concorda/seneca-concorda#7]
Since this is a seneca plugin, i think the correct name for it will be to rename it to 'seneca-concorda'.

This will allow the `seneca.use('concorda')` mapping to work, and be more clear about the nature of this repo.

**Comment by AdrianRossouw** [comment]
@mirceaalexandru @mcdonnelldean

**Comment by mirceaalexandru** [comment]
Yes, I think is OK to do this as this is - as you said - the seneca client.
